lms/djangoapps/courseware/views.py

```diff
@@ -1138,57 +1140,48 @@ def get_course_lti_endpoints(request, course_id):
 
 def get_analytics_answer_dist(request):
     """
-    Calls the the analytics answer distribution api. Retrieves answer distribution data for the in-line analytics display.
+    Calls the the analytics answer distribution api client. Retrieves answer distribution data for the in-line analytics display.
 
     Arguments:
         request (django request object):  the HTTP request object that triggered this view function
 
     Returns:
-        (django response object):  JSON response.  404 if api url is not found, 500 if server error, otherwise 200 with JSON body.
+        (django response object):  JSON response. 500 if error occurred, 404 if api client returns no data, otherwise 200 with JSON body.
     """
 
     all_data = json.loads(request.POST['data'])
     module_id = all_data['module_id']
     question_types_by_part = all_data['question_types_by_part']
     num_options_by_part = all_data['num_options_by_part']
-    course_key = SlashSeparatedCourseKey.from_deprecated_string(all_data['course_id'])
+    course_key = SlashSeparatedCourseKey.from_string(all_data['course_id'])
+
+    zendesk_base_url = getattr(settings, 'ZENDESK_URL')
+    zendesk_url_str = "<a href=\"" + zendesk_base_url + "/hc/en-us/requests/new\">here</a>"
 
     # Check user is enrolled as a staff member of this course
     try:
         course = get_course_with_access(request.user, 'staff', course_key, depth=None)
     except Http404:
-        return HttpResponseServerError(_('A problem has occurred retrieving the data, please report the problem.'))
+        return HttpResponseServerError(_("A problem has occurred retrieving the data, to report the problem click {url}").format(url=zendesk_url_str))
```

Comment:
This will be messy, but the `{url}` has the text "here" which should be included in translation. I think this needs to be split into `{zendesk_link_start}here{link_end}`